ena-submission/scripts/trigger_submission_to_ena.py

```diff
@@ -103,19 +103,25 @@ def trigger_submission_to_ena(
             return
 
     while True:
+        logger.debug("Checking for new sequences to upload to submission_table")
         # In a loop get approved sequences uploaded to Github and upload to submission_table
-        response = requests.get(
-            config.github_url,
-            timeout=10,
-        )
-
-        if response.ok:
+        try:
+            response = requests.get(
+                config.github_url,
+                timeout=10,
+            )
+            response.raise_for_status()
+        except requests.exceptions.RequestException as e:
+            logger.error(f"Failed to retrieve file due to requests exception: {e}")
+            time.sleep(min_between_github_requests * 60)
+            continue
+        try:
             sequences_to_upload = response.json()
```

Comment:
```suggestion
            sequences_to_upload = response.json()
```

I think you could move the sequences_to_upload outside of the try block?